src/main/java/seedu/dietmanager/DailyFoodRecord.java

```diff
@@ -68,13 +71,12 @@ public void showDailyRecord() {
     public String showBreakfast() {
         String message = "";
         for (Food food : breakfast) {
-            message = message + food.getFoodName() + ", ";
+            message = message + food.toString();
         }
         if (message.equals("")) {
-            message = "Oops, you have no record for this meal";
-            return message;
+            message = "Oops, you have no record for this meal.\n";
```

Comment:
Magic String, use Message Bank class instead

Reply:
ahahaha, okay, not written by me, but i can shift it to bank